agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3728,92 +3728,83 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             return false;
         }
 
-        auto vs = cmdu_tx.add_vs_tlv(ieee1905_1::tlvVendorSpecific::eVendorOUI::OUI_INTEL);
-        if (!vs) {
-            LOG(ERROR) << "Failed adding intel vendor specific TLV";
-            return false;
-        }
-
-        auto notification = message_com::add_intel_vs_data<
-            beerocks_message::cACTION_CONTROL_SLAVE_JOINED_NOTIFICATION>(cmdu_tx, vs);
+        auto vs_tlv =
```

Comment:
I would keep the name `notification`: it's more descriptive and more correct (because it is not actually a tlvVendorSpecific).

Reply:
ok will fix